packages/create-svelte/shared/+typescript/package.json

```diff
@@ -1,7 +1,12 @@
 {
+	"scripts": {
+		"check": "svelte-check --tsconfig ./tsconfig.json",
```

Comment:
Does `svelte-check` already supports `.ts` files, or should we also add `tsc --noEmit` here?

Reply:
Since version 1.6.0 it does also check ts/js files when using the `--tsconfig` flag